workinggroups/_posts/2015-11-04-licensing.md

```diff
@@ -3,8 +3,16 @@ title: Licensing
 layout: plain
 ---
 
-Providing recommendations and standards for software licensing has been identified as a high priority for the HSF. To this end the software licensing working group is being established.
+Since the early age of the HSF, software projects interested into participating mentioned the licensing as one important topic where the HSF could help. Many projects in our community don't have an explicit license, some have started with one license but are facing problems as the project is growing and incorporating new partners.
```

Comment:
"Since the early age" sounds strange. What about "Many of the projects interested in participating to the HSF mentioned licensing as in important topic where the HSF could help"